test/testcafe/spec/EnrollAuthenticatorOktaVerify_spec.js

```diff
@@ -0,0 +1,126 @@
+import xhrAuthenticatorEnrollOktaVerifyViaEmail from '../../../playground/mocks/data/idp/idx/authenticator-enroll-ov-via-email';
+import xhrAuthenticatorEnrollOktaVerifyEmail from '../../../playground/mocks/data/idp/idx/authenticator-enroll-ov-email';
+import xhrAuthenticatorEnrollOktaVerifyViaSMS from '../../../playground/mocks/data/idp/idx/authenticator-enroll-ov-via-sms';
+import xhrAuthenticatorEnrollOktaVerifySMS from '../../../playground/mocks/data/idp/idx/authenticator-enroll-ov-sms';
+import xhrSuccess from '../../../playground/mocks/data/idp/idx/success';
+
+const logger = RequestLogger(/poll/);
+
+const mock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrAuthenticatorEnrollOktaVerifyQr);
+
+const enrollViaQRcodeMocks = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrAuthenticatorEnrollOktaVerifyQr)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/poll')
+  .respond(xhrSuccess);
+
+const enrollViaEmailMocks = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrAuthenticatorEnrollOktaVerifyQr)
+  .onRequestTo('http://localhost:3000/idp/idx/credential/enroll')
+  .respond(xhrAuthenticatorEnrollOktaVerifyViaEmail)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/send')
+  .respond(xhrAuthenticatorEnrollOktaVerifyEmail)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/poll')
+  .respond(xhrSuccess);
+
+const enrollViaSmsMocks = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrAuthenticatorEnrollOktaVerifyQr)
+  .onRequestTo('http://localhost:3000/idp/idx/credential/enroll')
+  .respond(xhrAuthenticatorEnrollOktaVerifyViaSMS)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/send')
+  .respond(xhrAuthenticatorEnrollOktaVerifySMS)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/poll')
+  .respond(xhrSuccess);
+
+fixture('Enroll Okta Verify Authenticator');
+
+async function setup(t) {
+  const enrollOktaVerifyPage = new EnrollOktaVerifyPageObject(t);
+  await enrollOktaVerifyPage.navigateToPage();
+
+  const { log } = await t.getBrowserConsoleMessages();
+  await t.expect(log.length).eql(3);
+  await t.expect(log[0]).eql('===== playground widget ready event received =====');
+  await t.expect(log[1]).eql('===== playground widget afterRender event received =====');
+  await t.expect(JSON.parse(log[2])).eql({
+    controller: null,
+    formName: 'enroll-poll',
+    authenticatorType: 'app',
+  });
+
+  return enrollOktaVerifyPage;
+}
+
+test.requestHooks(logger, enrollViaQRcodeMocks)('should be able to enroll via qrcode', async t => {
+  const enrollOktaVerifyPage = await setup(t);
+  await t.expect(enrollOktaVerifyPage.getFormTitle()).eql('Set up Okta Verify');
+  await t.expect(enrollOktaVerifyPage.hasEnrollViaQRInstruction()).eql(true);
+  await t.expect(enrollOktaVerifyPage.hasQRcode()).eql(true);
```

Comment:
do you need to verify other details (instruction for set up qr code)  at this page